src/roomModes/ninja/module.lua

```diff
@@ -571,43 +619,106 @@ function resetAll()
-    --     resetAll()
-    -- end
+    if checkMod(playerName) == true then
+        isMod = true
+        isOp = true
+    end
 
-    if arg[1]=="m" and customroom == true and admin == playerName then
-        tfm.exec.newGame(arg[2])
-        resetAll()
+    for index, name in pairs(oplist) do
+        if name == playerName then
+            isOp = true
+        end
     end
 
-    if arg[1]=="m" then
-        tfm.exec.newGame(arg[2])
-        resetAll()
+    if admin == playerName and customroom == true then
+        isOp = true
+    end
+
+    -- OP ONLY ABILITIES (INCLUDES MOD)
+    if isOp == true then
+        if arg[1] == "m" then
+            if arg[2] ~= nil then
+                tfm.exec.newGame(arg[2])
+                mapstarttime = os.time()
+                resetAll()
+            end
+        end
+    end
+
+    -- MOD ONLY ABILITIES
+    if isMod == true then
+        if arg[1] == "mod" then
+            if checkRoomMod(playerName) == false then
+                table.insert(modroom, playerName)
+                local message = "You are a mod!"
+                print(message)
+                tfm.exec.chatMessage(message, playerName)
+            else
+                for index, name in pairs(modroom) do
+                    if name == playerName then
+                        table.remove(modroom, index)
+                        local message = "You are no longer a mod!"
+                        print(message)
+                        tfm.exec.chatMessage(message, playerName)
+                        break
+                    end
+                end
+            end
+            setColor(playerName)
+        end
+
+        if arg[1] == "n" then
+            mapwasskipped = true
+        end
+
+        if arg[1] == "op" then
+            if arg[2] ~= nil then
+                table.insert(oplist, arg[2])
+                local message = arg[2].." is an operator!"
+                print(arg[2].." is an operator!")
+                tfm.exec.chatMessage(message, playerName)
+            end
+        end
+
+        if arg[1] == "unop" then
+            if arg[2] ~= nil then
+                for index, name in pairs(oplist) do
+                    if name == arg[2] then
+                        table.remove(oplist, index)
+                        local message = arg[2].." is no longer an operator."
+                        print(arg[2].." is no longer an operator.")
+                        tfm.exec.chatMessage(message, playerName)
+                        break
+                    end
+                end
+            end
+        end
+
+        if arg[1] == "a" then
+            if arg[2] ~= nil then
+                for i = 3, #arg do
+                    arg[2] = arg[2].." "..arg[i]
+                end
+                local message = "\n<font color='#2E72CB'>Owner "..playerName..": "..arg[2].."</font>"
+                print(message)
+                tfm.exec.chatMessage(message)
+            end
+        end
```

Comment:
Discouraged system.

If possible make it visible that you're the owner of >ninja<

Reply:
Okay, i'll update that now